content/questions/copilot/question-002.md

```diff
@@ -0,0 +1,15 @@
+---
+archetype: "questions"
+title: "Question 002"
+question: "Which option below is NOT a possible way to grant access to Copilot for members of an organization?"
+draft: false
+---
+
+> Article Grant access to Copilot for members of an organization -https://docs.github.com/en/copilot/managing-copilot/managing-github-copilot-in-your-organization/managing-access-to-github-copilot-in-your-organization/granting-access-to-copilot-for-members-of-your-organization
+
+
+1. [x] As a member of an Organization, you can enable Copilot directly from your account settings.
+    > (optional) As a member you can request access but not directly enable your own access (when under an Organization)
```

Comment:
The way explanations work in this project -> if you select any INCORRECT answer then at the end of the exam you can see explanation why that answer is wrong (if there is an explanation, since they are optional)

Explanations for correct answers are never shown. So we can remove this one
```suggestion
1. [x] As a member of an Organization, you can enable Copilot directly from your account settings.
```

Reply:
Fixed 😄